src/sage/rings/polynomial/polynomial_singular_interface.py

```diff
@@ -189,7 +189,7 @@ def _singular_(self, singular=None):
 
         - ``singular`` -- Singular instance
 
-        OUTPUT: Singular ring matching this ring
+        OUTPUT: singular ring matching this ring
```

Comment:
```suggestion
        OUTPUT: Singular ring matching this ring
```

(for the Singular CAS)